source-salesforce/tests/snapshots/snapshots__spec__capture.stdout.json

```diff
@@ -4,42 +4,22 @@
       "$schema": "http://json-schema.org/draft-07/schema#",
       "additionalProperties": true,
       "properties": {
-        "credentials": {
-          "properties": {
-            "auth_type": {
-              "const": "Client",
-              "default": "Client",
-              "type": "string"
-            },
-            "client_id": {
-              "airbyte_secret": true,
-              "description": "Enter your Salesforce developer application's Client ID",
-              "order": 2,
-              "title": "Client ID",
-              "type": "string"
-            },
-            "client_secret": {
-              "airbyte_secret": true,
-              "description": "Enter your Salesforce developer application's Client secret",
-              "order": 3,
-              "title": "Client Secret",
-              "type": "string"
-            },
-            "refresh_token": {
-              "airbyte_secret": true,
-              "description": "Enter your application's Salesforce Refresh Token used to access your Salesforce account.",
-              "order": 4,
-              "title": "Refresh Token",
-              "type": "string"
-            }
-          },
-          "required": [
-            "client_id",
-            "client_secret",
-            "refresh_token"
-          ],
-          "type": "object",
-          "x-oauth2-provider": "salesforce"
+        "auth_type": {
```

Comment:
This schema change is ... wrong. I'm not sure what's happened here. Note that `auth_type` is top-level, but should be nested under `credentials` -- compare to spec.yaml.

Reply:
Yeah, it doesn't make sense, probably lost it in the rebase, will replace it.

Reply:
Corrected snapshot in [c6b4021](https://github.com/estuary/connectors/pull/1355/commits/c6b4021b93c9c5c798eac1bf37ece052c9c4cf5d)